pints/_mcmc/__init__.py

```diff
@@ -293,13 +293,60 @@ class MCMCController(object):
     method : class
         The class of :class:`MCMCSampler` to use. If no method is specified,
         :class:`HaarioBardenetACMC` is used.
+    transform : pints.Transform
+        A :class:`pints.Transform` that transform the model parameter space to
+        search space. If no transform is specified, search will be performed
+        in the model space.
     """
 
-    def __init__(self, log_pdf, chains, x0, sigma0=None, method=None):
+    def __init__(
+            self, log_pdf, chains, x0, sigma0=None, method=None,
+            transform=None):
 
-        # Store function
+        # Check function
         if not isinstance(log_pdf, pints.LogPDF):
             raise ValueError('Given function must extend pints.LogPDF')
+
+        # Transform everything
+        # From this point onward the MCMC sampler will see only the
+        # transformed search space and will know nothing about the model
+        # parameter space.
+        if transform:
+            log_pdf = transform.apply_log_pdf(log_pdf)
+            x0 = [transform.to_search(x) for x in x0]
+            #
+            # Following Eq. 5 in
```

Comment:
This is important information that should be in the main docstring for the controller

Reply:
It's moved to Transform now as `convert_covariance_matrix`.